### ISSUE TITLE
grunt test - error

### ISSUE THREAD

**Issue by eduardoinnorway** [leanovate/generator-cc-angular#4]
grunt test results in:

Running "protractor:e2e" (protractor) task
WARNING - pattern /Users/eduardo/IdeaProjects/generator-cc-angular/src/*_/_.feature did not match any files.

Did you forget to add some test .feature files for the latest release?

**Comment by AnBauer** [comment]
The first test feature file is generated together with the first partial created with cc-angular

**Comment by eduardoinnorway** [comment]
lol, its in the docs .... but then I did a partial and something else happend in "app.module.js" it added the "state" to the top of the file instead of adding it to $stateProvider:

   {
               state: 'app.dashboard',
               config: {
                       url: '/dashboard',
                       templateUrl: 'partial/dashboard/dashboard.html',
                       controller: 'DashboardCtrl',
                       controllerAs: 'DashboardCtrl',
                       title: 'Dashboard'
              }
        },
(function(angular) {
    'use strict';

```
/*
 * here is the right place to configure shared modules like app.appCore and feature-module
 * app.appCore defines the dependencies to angular and third party modules
 * any feature module should depend on app.appCore and possible other shared modules
 */

/**
 * @ngdoc: overview
 * @name: generatorCcAngular
 *
 */
angular.module('generatorCcAngular', ['appCore']);
```

})(angular);

**Comment by AnBauer** [comment]
ah. that is indeed a bug, I discovered today: you cannot add a partial to the root module, because the

```
function getStates() {
        return [

            /* Add New States Above */
        ];
    }
```

is not in the root module. I guess, I just did not think of partial bound to root. Gonna fix it in the next version.

**Comment by eduardoinnorway** [comment]
I worked a lot with cg-angular since 1 year backa with bigger applications and now I will implement cucumber tests or find a generator that follows the principals of cg-angular and also have cucumber tests. This is the reason I stumble upon your generator. I could be intersted to help out if you want more people in your team, I think there is some improvements to be done to cg-angular in general but your generator is more interesting coming that it also covers cucumber.

**Comment by AnBauer** [comment]
fixed in 1.0.3
